OpenKeychain/src/main/res/xml/preference_headers.xml

```diff
@@ -1,8 +1,11 @@
 <preference-headers xmlns:android="http://schemas.android.com/apk/res/android">
     <header
+        android:fragment="org.sufficientlysecure.keychain.ui.SettingsActivity$GuiPrefsFragment"
+        android:title="@string/section_gui" />
+    <header
         android:fragment="org.sufficientlysecure.keychain.ui.SettingsActivity$CloudSearchPrefsFragment"
         android:title="@string/section_cloud_search" />
     <header
         android:fragment="org.sufficientlysecure.keychain.ui.SettingsActivity$AdvancedPrefsFragment"
-        android:title="@string/section_passphrase_cache" />
```

Comment:
section_passphrase_cache is actually more accurate, all unrelated settings were removed from that dialog at some point